vllm/model_executor/layers/mamba/mamba_mixer.py

```diff
@@ -421,16 +431,25 @@ def split_batch_to_prefill_and_decode(
     num_decode_tokens: int,
     num_prefills: int,
     num_decodes: int,
+    num_padded_decodes: int,
 ) -> PrefillDecodeSplit:
+    num_actual_tokens = num_prefill_tokens + num_decode_tokens
```

Comment:
Just thinking this through: if FCG is enabled then `num_prefill_tokens=0`, but wouldn't we also want to use `num_decode_tokens` in this calculation (and this splits below)? Wouldn't the `hidden_states_BC` get padded up to the cuda graph batch size too?

Reply:
OK, I think that is the right way to do it but I'm also wondering if our tests are sufficient to cover what happens in these scenarios. I will look into that, but let's merge this PR first.